superset/explore/schemas.py

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+#  License ); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+#  AS IS  BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from marshmallow import fields, Schema
+
+
+class DatasetSchema(Schema):
+    cache_timeout = fields.Int()
+    column_formats = fields.Dict()
+    columns = fields.List(fields.Dict())
+    database = fields.Dict()
```

Comment:
Oh boy 😆. Some of them are dynamic and others have really extensive trees 😞. It's also hard to check all the possible values because many are defined as dictionaries. For this PR, I limited the depth to two levels and tried to map the attributes the best I could.

Reply:
it's a pretty great improvement already